04_setup_ironic.sh

```diff
@@ -49,6 +49,14 @@ for IMAGE_VAR in $(env | grep "_LOCAL_IMAGE=" | grep -o "^[^=]*") ; do
     echo "RUN sed -i 's%$OLDIMAGE%${!IMAGE_VAR}%g' /release-manifests/*" >> $DOCKERFILE
 done
 
+if [ ! -z "${MIRROR_IMAGES}" ]; then
```

Comment:
By default this is undefined, probably we should add it defaulted to an empty string in `common.sh`, or else default it everywhere like `${MIRROR_IMAGES:-}`

Reply:
Done